tutorials/tour/_posts/2017-02-13-unified-types.md

```diff
@@ -48,3 +48,15 @@ c
 true
 <function>
 ```
+
+## Nothing and Null
+`Nothing` is a subtype of all types. There is no value that has type `Nothing`. There are two uses: The first is to signal abnormal termination such as an exception (i.e. it doesn't return). The second type usage is an element type of empty collections (e.g. `Set[Nothing]`).
+
+`Null` is a subtype of all subtypes of `AnyRef`. The class name is `Null` but the literal identifier is `null`. You can assign any reference type to null.
```

Comment:
A few suggested edits:

`Null` is a subtype of all reference types (i.e., subtypes of `AnyRef`). It has a single value identified by the keyword literal `null`. You can assign any reference type to null.

It might also be helpful to note that null is provided mostly for interoperability with other JVM languages and is not commonly used in Scala code.